CHANGELOG.md

```diff
@@ -33,6 +33,7 @@
 - Added support for `Series.dt.dayofweek`, `Series.dt.day_of_week`, `Series.dt.dayofyear`, and `Series.dt.day_of_year`.
 - Added support for `Series.str.__getitem__` (`Series.str[...]`).
 - Added support for `Series.str.lstrip` and `Series.str.rstrip`.
+- Added support for `DataFrameGroupby.size`.
```

Comment:
should we support SeriesGroupby.size too?

Reply:
I wasn't planning to, but it wasn't too much more effort 😄